lib/navigation/imu_displacement.hpp

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include <array>
+#include <cstdint>
+#include <optional>
```

Comment:
are you using this?

Reply:
Not currently but will probably be implementing some sort of fail state